app/main.py

```diff
@@ -15,5 +15,8 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+sorted_variables = {
+    "mutable" : ["my_favourite_films", "marks", "collection_of_coins"],
```

Comment:
The `collection_of_coins` variable is categorized as mutable, but sets in Python are actually mutable. Consider moving `collection_of_coins` to the 'immutable' category if you intended to categorize it as such.